### DISCUSSION TITLE
How to change the language of a user?

### DISCUSSION THREAD

**Discussion by msharp1** [datenstrom/community#122] [Ask a question] [answered]
I've created the file `language-fr.ini` (with the provided fr yellow language file) and updated `config.ini` to `Language: fr`, but I still get the english text labels. What could I miss ?

Followed the instructions found at https://github.com/datenstrom/yellow-extensions/tree/master/languages/fran%C3%A7ais

**Comment by nibreh** [comment, marked as answer]
En fait ce n'est pas indiqué dans le README.md, mais tu dois configurer le fichier `user.ini` avec `fr`, et voilà !

You must set the language in the `user.ini` - thx mark ;)

**Comment by msharp1** [comment]
and thanks ! :) (Maybe the documentation should be updated ?)
